tests/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM rust:latest
+
+# Install Node.js
+ENV NODE_VERSION 12.0.0
+RUN curl https://raw.githubusercontent.com/creationix/nvm/master/install.sh | bash \
```

Comment:
I think we might have to include all the different languages ( python, Go etc.) here? Could get pretty intense?

Reply:
It definitely can get hairy.
I'd be curious to know if we can find a good polyglot image with fixed versions, but also the ability to install stable Rust. 🤔